README.md

```diff
@@ -398,7 +398,7 @@ expect {
 I expected subject: () -> kotlin.Nothing        (readme.examples.MostExamplesSpec$1$6$1 <1234789>)
 ◆ ▶ thrown exception when called: java.lang.IllegalArgumentException
     ◾ ▶ message: null
-        ◾ to be an instance of type: String (kotlin.String) -- Class: java.lang.String
+        ◾ not to equal: null but to be an instance of type: String (kotlin.String) -- Class: java.lang.String
```

Comment:
don't forget to run `./gradlew :readme-examples:build`  this should change if you adopt the translation